infrastructure/base/modules/env/main.tf

```diff
@@ -147,6 +147,10 @@ module "backend_cloudrun" {
       name  = "DATABASE_HOST"
       value = module.database.database_host
     },
+    {
```

Comment:
this should not be needed: https://cloud.google.com/ruby/docs/reference/google-cloud-translate/latest/AUTHENTICATION